java/src/apps/GuiLafConfigPane.java

```diff
@@ -212,22 +184,29 @@ private int getDefaultFontSize() {
     static ActionListener listener;
 
     public void doFontSize(JPanel panel) {
-
+        GuiLafPreferencesManager manager = InstanceManager.getDefault(GuiLafPreferencesManager.class);
```

Comment:
This makes an assumption that the manager provided by InstanceManager will always be the same, which is not guaranteed. It would be better to provide a convince method that always requests the default GuiLafPreferencesManger from the InstanceManager if you do not want a shorter way to get the manager.